eq-author/src/components/Forms/WrappingInput/index.js

```diff
@@ -48,14 +50,19 @@ class WrappingInput extends React.Component {
   };
 
   render() {
-    const { bold, placeholder, ...otherProps } = this.props;
+    const { bold, placeholder, errorValidationMsg, ...otherProps } = this.props;
     return (
       <StyleContext
         bold={bold}
         onChange={this.handleChange}
         onKeyDown={this.handleKeyDown}
       >
-        <TextArea {...otherProps} placeholder={placeholder} />
+        <TextArea
+          {...otherProps}
+          placeholder={placeholder}
+          invalid={errorValidationMsg ? "true" : null}
```

Comment:
Can we use booleans here?

Reply:
I tried that but got a warning
``
index.js:1384 Warning: Received 'false' for a non-boolean attribute 'invalid'.

If you want to write it to the DOM, pass a string instead: invalid="false" or invalid={value.toString()}.

If you used to conditionally omit it with invalid={condition && value}, pass invalid={condition ? value : undefined} instead.
``

I'll change it so it just takes the errorValidationMsg string.

Reply:
The issue is styled components is passing this onto the DOM. We should solve that problem instead by wrapping the `TextArea` in another component that only passes the valid props on:

https://github.com/ONSdigital/eq-author-app/compare/prototype-survey-intro?expand=1#diff-02621415b811b4593f1d08f24d9b9c81R16

Reply:
Changed to use wrapper and boolean prop as suggested